src/Scripting/CSharp/Hosting/CommandLine/Csi.cs

```diff
@@ -18,15 +18,21 @@ internal CSharpInteractiveCompiler(string responseFile, string baseDirectory, st
         {
         }
 
+        internal override Type Type => typeof(CSharpInteractiveCompiler);
+
+        internal override string GetAssemblyFileVersion()
+        {
+            return Type.GetTypeInfo().Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>().Version;
```

Comment:
Why not do this everywhere rather than having this be overridden? It seems like this should work in all cases?